src/test/suite/index.ts

```diff
@@ -1,38 +1,50 @@
+'use strict';
+
 import * as path from 'path';
 import * as Mocha from 'mocha';
 import * as glob from 'glob';
 
-export function run(): Promise<void> {
-	// Create the mocha test
-	const mocha = new Mocha({
-		ui: 'tdd',
-		color: true
+// The test coverage approach is inspired by https://github.com/microsoft/vscode-js-debug/blob/master/src/test/testRunner.ts
+function setupCoverage() {
+    const NYC = require('nyc');
+	const nyc = new NYC({
```

Comment:
Formatting looks rather weird

Reply:
might be for another time with prettier tho

Reply:
I agree this is wonky. `.editorconfig` says the following:

```
indent_style = tab
indent_size = 4
```

I'm not sure why that's not being respected. I also don't agree with these defaults, but that's not of much importance if this is remarkjs or unifiedjs defaults. Are they? No matter what they are, I'll be sure to reformat all files according to the specification.

Reply:
They’re not my defaults. There is no unified style, but I originally made most of the unified projects so that’s used a lot. I’m all for sticking with project guidelines, instead of imposing my styles though, and not discussing it too much.
We can change these styles (later) as we all want it to be something else.